zappa/zappa.py

```diff
@@ -792,13 +784,13 @@ def create_and_setup_methods(self, api_id, resource_id, lambda_arn, report_progr
                 response_parameters = {"method.response.header." + header_type: False for header_type in self.method_header_types}
                 response_models = {content_type: 'Empty' for content_type in self.method_content_types}
 
-                method_response = self.apigateway_client.put_method_response(
-                        restApiId=api_id,
-                        resourceId=resource_id,
-                        httpMethod=method,
-                        statusCode=status_code,
-                        responseParameters=response_parameters,
-                        responseModels=response_models
+                self.apigateway_client.put_method_response(  # method_response = unused
```

Comment:
yes, I just moved unused variables to comments with = character, so you can see this variable was unused and removed. To ensure it just was forgotten/wrongly assigned and no hidden bugs here.